xbmc/windowing/osx/WinEventsOSXImpl.mm

```diff
@@ -196,19 +196,8 @@ - (void)enableInputEvents
 {
   [self disableInputEvents]; // allow only one registration at a time
 
-  // clang-format off
-  // Create an event tap. We are interested in mouse and keyboard events.
-  NSEventMask eventMask =
-      NSEventMaskKeyDown | NSEventMaskKeyUp |
-      NSEventMaskLeftMouseDown | NSEventMaskLeftMouseUp |
-      NSEventMaskRightMouseDown | NSEventMaskRightMouseUp |
-      NSEventMaskOtherMouseDown | NSEventMaskOtherMouseUp |
-      NSEventMaskScrollWheel |
-      NSEventMaskLeftMouseDragged |
-      NSEventMaskRightMouseDragged |
-      NSEventMaskOtherMouseDragged |
-      NSEventMaskMouseMoved;
-  // clang-format on
+  // Create an event tap. We are interested keyboard events (mouse is sent from the trackingArea/NSView).
```

Comment:
The mouse actions (ie button up/down/dragged etc) have nothing to do with the NSTrackingArea. Its entirely NSWindow/NSView inheritance of NSResponder

https://developer.apple.com/documentation/appkit/nsresponder?language=objc

Just to not muddy the waters for people who come later.

Reply:
hum so maybe we should only keep mouse moved in NSView and keep all the other actions as they were before? This is pretty much the only event that we're interesting in to be honest.

Reply:
well to be honest, id almost go the other way. move the key up/down to the view handler, and drop the universal tap.

I didnt mention that initially because im not sure if/what any regressions might be in key handling (super key combos and what not). If it aint broke, dont fix it kinda thing.

Reply:
thanks for the feedback!
Gave it a shot and works (after reimplementing acceptsFirstResponder) 👍 
No issues with the shortcuts too (both kodi and macos)